intranet/static/js/common.header.js

```diff
@@ -78,4 +84,4 @@ $(function() {
 
     arrowPosition();
     window.onresize = arrowPosition;
-});
+});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2747560293/issues/source?bid=3176160&fileBranchId=3355598#l87)